client/assets/calendar.js

```diff
@@ -0,0 +1,121 @@
+import { saveAs } from 'file-saver'
+import Moment from 'moment'
+import {
+  extendMoment
+} from 'moment-range'
+
+export async function downloadCalendar({ plan }) {
+	const FileSaver = require('file-saver')
+	const moment = extendMoment(Moment)
+	const ics_day_list = ['SU', 'MO', 'TU', 'WE', 'TH', 'FR', 'SA'] //days of the week for ICS standard
+	const semester_year = (plan.courses[0].dates[0].slice(0, 2) < moment().month()) ? moment().year() + 1 : moment().year()
+
+	//required iCalendar info
+	//calendar_string is an array because what we use to print requires an array
+	let calendar_string = ['BEGIN:VCALENDAR\nPRODID:-//Nick and Pedram, Squaaaad//Course Calendar//EN\nVERSION:2.0\nCALSCALE:GREGORIAN']
```

Comment:
First, you define it to be an array then start appending to it making it into a string. Just make it a string.

Also to not have to do all this constant appending look into [template strings in JavaScript](http://lmgtfy.com/?q=template+strings)